lib/ajax-requests/all_foods.js

```diff
@@ -0,0 +1,24 @@
+//
+const $ = require('jquery')
+
+const url = "https://quantified-self-aabs.herokuapp.com/api/v1/foods"
+// let foodsJson = getAllFoods();
+
+function getAllFoods() {
+    $.ajax({
```

Comment:
I think technically this one probably doesn't need to be ajax. Not sure it matters though.